Makefile

```diff
@@ -44,14 +44,27 @@ integration-test-sumcoins:
 		--db-address "$(STANDARD_REDIS_ADDR)" --db-slot "$(STANDARD_REDIS_DB)" \
 		--encoding "$(STANDARD_ENCODING_TYPE)"
 
-integration-test-sumcoins-python:
+
```

Comment:
given that we just want it for the integration test, I would just turn it into a shell script or python script and also checkin the generated file, only updating it via that script if needed to be updated

Reply:
For that case, yes makes sense (y)